src/main/webapp/event.html

```diff
@@ -14,7 +21,29 @@ <h2>template</h2>
   Recruit volunteer or event:<br>
   <input type="text" name="volunteer" maxlength = "25" size="100" placeholder="Please enter TRUE for volunteer FALSE for event">
   <br><br>
-  <input type="submit" value="Submit">
 </form> 
+<form id="asd">
+  Name:<br>
+  <input type="text" maxlength = "25" size="100" name="Last Name  ">
+  <br>
+  Why want to join the event:<br>
+  <input type="text" name="why" maxlength = "25" size="100" placeholder="Why want to join the event">
+  <br>
+  Past experience:<br>
+  <input type="text" name="experience" maxlength = "25" size="100" placeholder="Past experience">
+  <br><br>
+</form>
+  <script>
```

Comment:
Should move this into a separate js file or add TODO comment so that you remember to do it in a follow-up.